www/js/app.js

```diff
@@ -167,7 +167,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     document.getElementById('openExternalLinksInNewTabsCheck').checked = params.openExternalLinksInNewTabs;
     switchHomeKeyToFocusSearchBar();
     document.getElementById('bypassAppCacheCheck').checked = !params.appCache;
-    document.getElementById('appVersion').innerHTML = 'Kiwix ' + params.appVersion;
+    document.getElementById('appVersion').appendChild(document.createTextNode('Kiwix ' + params.appVersion)); 
```

Comment:
This works, but why did you use a different way for this one?

Reply:
I googled and it was showing that appenchild was one of the alternatives to using innerHTML so I did this

Reply:
> This works, but why did you use a different way for this one?

Okay, I changed it like the other areas where I used textContent and it works
I was experimenting before to find out which one of them worksso this one was different